generate_training_data_scripts/makejson.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+import json
+
+
+def makejson(obj, eng):
+    '''
+    title         = obj.extra_data['source_data']['data']['titles'][0]['title']
+    abstracts     = obj.extra_data['source_data']['data']['abstracts'][0]['value']
+    authors_dict  = obj.extra_data['source_data']['data']['authors']
+    arxiv_eprints = obj.extra_data['source_data']['data']['arxiv_eprints']
+    preprint_date = obj.extra_data['source_data']['data']['preprint_date']
+
+
+
+    refs = obj.data['references']
+    authors_list = []
+    for i in range(len(authors_dict)):
+        a_d = authors_dict[i]
+        authors_list.append(a_d["full_name"])
+
+    for i in range(len(refs)):
+        r = refs[i]["reference"]
+        ref_authors_list = []
+        ri_authors = r["authors"]
+        for j in range(len(ri_authors)):
+            r_auth_dict = ri_authors[j]
+            ref_authors_list.append(r_auth_dict["full_name"])
+
+        r["authors"] = ref_authors_list
+
+
+    object_data = {"title": title, "abstracts": abstracts, "authors": authors_list,
+                    "arxiv_eprints": arxiv_eprints, "preprint_date": preprint_date, "references": refs}
```

Comment:
I think you can delete everything in the docstring here since I believe it's just old code that the British guys were using before they found an easier way